backend/src/main.rs

```diff
@@ -151,11 +157,14 @@ async fn init_state(cli_args: &Args) -> Result<AppState> {
     );
 
     Ok(AppState {
-        config,
+        config: config.clone(),
```

Comment:
```suggestion
        config: &CONFIG,
```
This will require updating the type of the `config` field, likely to an `&'static AppConfig`